agent/provider/src/cli/rule.rs

```diff
@@ -37,6 +71,15 @@ pub enum SetOutboundRule {
     Partner(PartnerRuleWithCert),
 }
 
+#[derive(StructOpt, Clone, Debug)]
+pub enum RestrictRuleDesc {
+    ByNodeId {
+        #[structopt(short, long)]
+        address: NodeId,
+    },
+    Certified(RestrictRuleWithCert),
+}
```

Comment:
Maybe we could want to unify it, because now it looks like this:
`rule add allow-only certified cert-id  cefd51ab`
`rule add allow-only certified import-cert ../golem-certificate/cert.new_key.signed.json`
but for `by-node-id`
`rule add allow-only by-node-id --address  0x9acce8db6bdbcae218be2ecf15175ca3c5f227fa`

Maybe it would be better to have
`rule add allow-only certified --cert-id  cefd51ab`
`rule add allow-only certified --cert-file ../golem-certificate/cert.new_key.signed.json`
?

Reply:
I will try to get rid of `--address`